roles/webview/templates/var/www/webview/scripts/settings.js

```diff
@@ -18,48 +22,33 @@
         host: location.hostname
       },
 
-      // Prefix to prepend to page titles
-      titleSuffix: ' - OpenStax CNX',
-
-      // Google Analytics tracking ID
-      analyticsID: 'UA-7903479-1',
-
-      // Supported languages
-      languages: languages,
-
       // Legacy URL
       // URLs are concatenated using the following logic: location.protocol + '//' + legacy + '/' + view.url
       //   Example: 'http:' + '//' + 'cnx.org' + '/' + 'contents'
       // Do not include the protocol or a trailing slash
       legacy: '{{ zope_domain }}',
 
       // Webmaster E-mail address
-      webmaster: 'cnx@cnx.org',
+      webmaster: 'support@openstax.org',
 
       // Content shortcodes
-      shortcodes: {
-      },
+      shortcodes: {{ content_shortcodes|default({})|to_json }},
```

Comment:
The existing variable is `concept_coach_webview_settings`. see also, https://github.com/Connexions/cnx-deploy/blob/master/environments/dev/group_vars/all/vars.yml#L55